dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -74,6 +80,20 @@ protected void service(final HttpServletRequest request, final HttpServletRespon
     }
   }
 
+  
+    private Optional<String> getParameter(final String uri, final String paramName) {
+
+        int start  = uri.indexOf("/" + paramName + "/") + paramName.length() + 2;
+        int end  = uri.indexOf("/", start) >-1 ? uri.indexOf("/", start) : uri.length();
+        
+        
+        return Optional.ofNullable(uri.indexOf("/" + paramName + "/") > 0 
+                        ? uri.substring(start,end) 
+                                        : null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)